src/ol/source/tilearcgisrestsource.js

```diff
@@ -109,6 +109,7 @@ ol.source.TileArcGISRest.prototype.getRequestUrl_ =
   params['BBOX'] = tileExtent.join(',');
   params['BBOXSR'] = srid;
   params['IMAGESR'] = srid;
+  params['DPI'] = 90 * pixelRatio;
```

Comment:
hey @ahocevar one question on the value, is 90 used because of the OGC SLD value of 90.71?
I see the default if not specified is 96 on ArcGIS REST http://sampleserver1.arcgisonline.com/ArcGIS/SDK/REST/export.html